### ISSUE TITLE
Allow the user to use raw text in "Equation/Result View" modes

### ISSUE THREAD

**Issue by vinmir** [mkulesh/microMathematics#109]
Hi.
I don't know if this is the case for every Android device, but I have some issues regarding text input in the two aforementioned modes. I'll use three screenshots I've taken to explain myself more clearly.

Consider the following situation: I want to input the expression `sin(9-1) + 6` in *Result View*.

First off, after inputting the first part of the expression -- `sin(9-1)` --, my text selection bar will refuse to leave the region bounded by the pair of parentheses, as illustrated below:
![mm1-rev-2](https://user-images.githubusercontent.com/8193958/110214577-f91a5480-7e83-11eb-8445-40b89d29f4b3.jpg)

In order to circumvent the issue, I have to first select the entire `sin(9-1)`section and then press the `[ ] + [ ]` button in the tool bar, as shown below:
![mm4-rev](https://user-images.githubusercontent.com/8193958/110215026-56afa080-7e86-11eb-8043-850da7013982.jpg)

Finally, I am once again able to move the text selection bar in my virtual keyboard and get the result I was looking for, which could very easily be inserted in the box via raw text:
![mm3-rev](https://user-images.githubusercontent.com/8193958/110215077-96768800-7e86-11eb-81bd-45501d14f95f.jpg)

As the left box of the *Equation* item already works in a raw text fashion, is there any way to give users the option of using raw text instead of the sudo-LaTeX implementation we have now for the other fields/boxes? As you can see, it's quite a hassle to get things to work whenever you want to use more complicated expressions, such as a custom *binomial coefficient of (m,n)* function. This is a great application and the only issue I have with it is this one.

Thanks.

**Comment by yashpalgoyal1304** [comment]
yeah, i was also having similar difficulty. i am pretty sure there is a pretty solid reason behind it - easy creation of the source for the equations.

but yeah, if possible, then i would like some improvements as well 😜
